components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -84,6 +86,12 @@ exec $HAB_ROOT_PATH/bin/hab pkg exec core/hab-plan-build hab-plan-build \$*
 EOF
   $bb chmod $v 755 $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/build
 
+  $bb cat <<TAIL_SUP > $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/tail_sup
```

Comment:
I think this is fine for now but somebody should open an issue to replace this with `hab sup logs -f`.

Since this is for studio use, I'm not concerned about the growth of this log file, but if habitat starts producing logs anywhere but stdout/stderr or the system journal, it has a responsibility to rotate them.